status-client-spec.md

```diff
@@ -216,9 +222,7 @@ see [Status EIPs Standards](status-EIPs.md).
 
 ## Security Considerations
 
-TBD.
-
-<!-- TODO: Fill this out. -->
+See [Appendix A](#appendix-a-security-considerations)
```

Comment:
This link seems weird?

Reply:
why? it just goes to the appendix at the bottom of the page

Reply:
ah, my bad, this is middle